assets/evaluation_on_cloud/environments/evaluations-built-in/context/online_eval/eval_common/_ua_patcher.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Patch User Agent in azure.ai.evaluation."""
+import inspect
+
+from azure.ai.evaluation import _user_agent
+
+
+def get_eval_type():
+    """Get evaluation type."""
+    full_stack = inspect.stack()
```

Comment:
do we have any better way to extract remote vs online evaluation trigger? Wouldn't reading through the entire call trace be computationally expensive?